jenkins_jobs_active_choice/active_choice.py

```diff
@@ -13,29 +13,23 @@
 
 import xml.etree.ElementTree as Xml
 
-
-CHOICE_TYPE = {
-    'single': 'PT_SINGLE_SELECT',
-    'multi': 'PT_MULTI_SELECT',
-    'checkbox': 'PT_CHECKBOX',
-    'radio': 'PT_RADIO',
-}
-
-REQUIRED = [
-    # (yaml tag)
-    ('name', 'name'),
-    ('project', 'projectName'),
+# these are common tags for both cascade-choice and dynamic-reference
+SCRIPT_OPTIONAL = [
+    # ( yaml tag, xml tag, default value )
+    ('script-sandbox', 'sandbox', 'False'),
+    ('script-classpath', 'classpath', 'False'),
 ]
 
-OPTIONAL = [
+FALLBACK_OPTIONAL = [
     # ( yaml tag, xml tag, default value )
-    ('description', 'description', ''),
-    ('visible-item-count', 'visibleItemCount', 1),
-    ('reference', 'referencedParameters', ''),
-    ('filterable', 'filterable', False),
+    ('fallback-sandbox', 'sandbox', 'False'),
+    ('fallback-classpath', 'classpath', 'False'),
 ]
 
 
+# use logging from JJB
+import logging
```

Comment:
import should be at begin of file

Reply:
Hi Bulat,

The import statement is now grouped with the others and the merge conflicts have been resolved.

Thanks!

- Allen